Modules/Loadable/Markups/Testing/Python/MarkupsCurveMeasurementsTest.py

```diff
@@ -25,24 +25,24 @@
 
 # Check number of arrays in the curve node
 curvePointData = curveNode.GetCurveWorld().GetPointData()
-if curvePointData.GetNumberOfArrays() != 1:
-    exceptionMessage = f"Unexpected number of data arrays in curve: {curvePointData.GetNumberOfArrays()} (expected 1)"
+if curvePointData.GetNumberOfArrays() != 4:  # PedigreeIDs Tangents Normals Binormals
+    exceptionMessage = f"Unexpected number of data arrays in curve: {curvePointData.GetNumberOfArrays()} (expected 4)"
     raise Exception(exceptionMessage)
 
 # Turn on curvature calculation in curve node
 curveNode.GetMeasurement("curvature max").SetEnabled(True)
 
 # Check curvature computation result
 curvePointData = curveNode.GetCurveWorld().GetPointData()
-if curvePointData.GetNumberOfArrays() != 2:
-    exceptionMessage = f"Unexpected number of data arrays in curve: {curvePointData.GetNumberOfArrays()} (expected 2)"
+if curvePointData.GetNumberOfArrays() != 5:
```

Comment:
It was useful that you listed the names of expected arrays above, could you add a comment about what are those 5 expected arrays?